src/Comparator.php

```diff
@@ -0,0 +1,104 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\ApiCompare;
+
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflection\ReflectionMethod;
+use Roave\BetterReflection\Reflection\ReflectionParameter;
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\Reflector\Exception\IdentifierNotFound;
+
+final class Comparator
+{
+    public function compare(ClassReflector $oldApi, ClassReflector $newApi): array
```

Comment:
Do we only compare classes?

Reply:
Return type `array` is too limited

Reply:
For now yes, if you want to support functions, add an issue :P